content/v1.9/_index.md

```diff
@@ -4,6 +4,7 @@ weight: 1
 toc_include: false
 aliases:
     - /docs
+    - /docs/v1.9
```

Comment:
Can we just put these literally under `/docs`?

Reply:
Yes, but it has to be through duplication. I'm closing this PR to submit another one with a different fix.